pages/store/claim.vue

```diff
@@ -69,6 +69,29 @@ const cartId = computed(() => getRouteQuery('cart_id'))
 const claimingToken = computed(() => getRouteQuery('claiming_token'))
 const paymentId = computed(() => getRouteQuery('payment_id'))
 
+const loadingLabels = [
+  $t('claim_page_loading_step_1'),
+  $t('claim_page_loading_step_2'),
+  $t('claim_page_loading_step_3'),
+]
```

Comment:
need to use computed here, as it wouldn't update when language changed

Reply:
[🐛 Update labels to use computed property](https://github.com/likecoin/3ook-com/pull/269/commits/f184a5badf8d159d1453f80d7758371c7b5d4293)